src/content/general/tools/mek/mozzarilla/readme.md

```diff
@@ -13,7 +13,7 @@ keywords:
 ---
 Mozzarilla is a file editing program, and is intended to be a complete replacement for Halo Custom Edition's Guerilla, with additional tools and ease of use upgrades, including:
 
-* Edits regular [tags][], [OpenSauce][] tags, and gametypes
+* Edits regular [tags][h1/tags], [OpenSauce][] tags, and gametypes
```

Comment:
Note that this page is under the `/general` tree so needs to disambiguate what tags it's talking about. I just picked one for now.